recipes/sphinxcontrib-spelling/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sphinxcontrib-spelling" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "008ec060f312367222992824abed00124cce0dd31c375a89b2053010df0e3be8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr
+  run:
+    - python    
+
+test:
+  imports:
+    - sphinxcontrib
+
+about:
+  home: https://bitbucket.org/dhellmann/sphinxcontrib-spelling
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'This package contains sphinxcontrb.spelling, a spelling checker for Sphinx-based documentation.'
+  description: |
+    This package contains sphinxcontrb.spelling, a spelling checker 
+    for Sphinx-based documentation. It uses PyEnchant to produce 
+    a report showing misspelled words.
+  doc_url: https://bitbucket.org/dhellmann/sphinxcontrib-spelling
+  dev_url: https://bitbucket.org/dhellmann/sphinxcontrib-spelling
+
+extra:
+  recipe-maintainers:
+    - cshaley
```

Comment:
@cshaley, are you ok being a maintainer?

Reply:
Yes